yt/fields/tests/test_fields_plugins.py

```diff
@@ -1,10 +1,22 @@
 import os
+import shutil
+import tempfile
 import unittest
 
 import yt
-from yt.config import CONFIG_DIR, ytcfg
+from yt.config import YTConfig, config_dir, ytcfg
 from yt.testing import assert_raises, fake_random_ds
 
+XDG_CONFIG_HOME = os.environ.get("XDG_CONFIG_HOME")
```

Comment:
why isn't this part of the test setup ?

Reply:
What's the difference? It's not going to be imported by anything

Reply:
You're right, but in this states it looks like it's going to be used in more than one test (which would make it a candidate for fixture in pytest), but it's not the case, so might as well define it in the only scope it used.